source/actions/get-images.js

```diff
@@ -11,16 +11,17 @@ const {
   promptConditionMatch,
   argumentsPrompt,
   taskExecutor,
-} = require('../helpers');
-const {
+} from '../helpers';
+
+import {
   DEFAULT_SAVE_PATH,
   PATH_TO_IMAGE,
   ORIENTATION_ALL,
   IMAGE_NAME_FORMAT_ORIGIN,
-} = require('../constants');
+} from '../constants';
 
 /* Action that handle extracting lock screen from windows */
-module.exports = async function (args, options, logger) {
+export default async function (args, options, logger) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function has a complexity of 10. Maximum allowed is 4.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4960498)